packages/dbgpt-app/src/dbgpt_app/openapi/api_v1/subagent/dispatcher.py

```
@@ -0,0 +1,739 @@
+                    ),
+                    timeout=_SUBAGENT_TIMEOUT,
+                )
+                r = extract_subagent_result(reply, sub_state, title, "done")
+            except asyncio.TimeoutError:
+                r = {
+                    "title": title,
+                    "status": "timeout",
+                    "result": f"子任务超时（{_SUBAGENT_TIMEOUT}s）",
+                    "artifacts": [],
+                }
+            except Exception as e:  # single failure must not break the batch
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)